content/src/content/jcr_root/apps/core/wcm/components/page/v2/page/clientlibs/site/digitalData/js/digitalData.js

```diff
@@ -0,0 +1,146 @@
+/*
+ *  Copyright 2016 Adobe Systems Incorporated
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+ window.digitalData = window.digitalData || {};
```

Comment:
This file fails the eslint build, [here's a fixed version](https://gist.github.com/bstopp/06f20879b5969e6fb60144a0907a6a62)